export const errors = {
    ErrNoData: new Error("нет данных"),
    ErrInternalError: new Error("произошла внутреняя ошибка"),
}